src/app/ApplicationTemplate.Shared.Views/ApplicationTemplate.Shared.Views.projitems

```diff
@@ -9,6 +9,9 @@
     <Import_RootNamespace>ApplicationTemplate.Views</Import_RootNamespace>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)Behavior\FormattingTextBoxBehavior.Android.cs" />
```

Comment:
I don't think we should be adding this behavior to the template. You don't actually need it for your validation features.

Reply:
But it make the sample more realistic for Phone Number and Postal Code fields

Reply:
Added comment in xaml to suggest implementing some form of text formating, this is a bit out of scope for the feature.